zooniverse/front-end-monorepo
# Refactor StepNavigation and shared components

## Package
- lib-classifier

## Linked Issue and/or Talk Post
- closes #3532 and #3190 

## Describe your changes
- create shared components alias
- move StepNavigation component to shared components
- fix ^ related imports and translations
- refactor StepNavigation options ids to fix #3532 
- refactor StepNavigation tests with `react-testing-library`

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
  - bug: https://frontend.preview.zooniverse.org/projects/markb-panoptes/digileap-testing-staging/classify/workflow/3594?env=staging
  - local with fix: https://local.zooniverse.org:3000/projects/markb-panoptes/digileap-testing-staging/classify/workflow/3594
- What user actions should my reviewer step through to review this PR?
  - click Tutorial (from within task area, not modal)
  - click Tutorial radio input, note (bug) subject toggles from image to text or (local/fix) tutorial advances and no changes to subject
- Which storybook stories should be reviewed? N/A, bug linked across two different components
- Are there plans for follow up PR’s to further fix this bug or develop this feature? no

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Bug Fix
